CONTRIBUTING.rst

```diff
@@ -100,14 +98,14 @@ Ready to contribute? Here's how to set up `dj-stripe` for local development.
 6. When you're done making changes, check that your changes pass the tests.
    A quick test run can be done as follows::
 
-   $ DJSTRIPE_TEST_DB_VENDOR=sqlite pytest --reuse-db
+   $ DJSTRIPE_TEST_DB_VENDOR=sqlite poetry run pytest --reuse-db
+
 
    You should also check that the tests pass with other python and Django versions with tox.
    pytest will output both command line and html coverage statistics and will warn you
    if your changes caused code coverage to drop.::
 
-    $ pip install tox
-    $ tox
+    $ poetry run tox
```

Comment:
Actually I think that's wrong, but I need to check (I don't think we're intending to run tox inside of poetry).